src/main/java/transfer/countrytrans/routers/country/FromFile.java

```diff
@@ -0,0 +1,18 @@
+package transfer.countrytrans.routers.country;
+
+import dto.countryservisedto.CountryResponseDTO;
+import transfer.ExpectedObjectFromFile;
+import util.FileReaderUtil;
+
+import java.io.BufferedReader;
+
+
+public class FromFile {
+
+    public ExpectedObjectFromFile<CountryResponseDTO> get(String filePath) {
```

Comment:
This class shouldn`t be in transfer because it`s not get request on api. It`s just method to get object from json and it`s not related to rest assured.